### PR TITLE
updated Tech Lead name,Current developers names,updated the content f…

### PR DESCRIPTION
**Pull request by BollepalliPranitha**
Changes Made: Updated the Tech lead and current developers names and Profile page information . Added Additional details about project's functionalities and benefits

What was changed:
Updated the tech lead name to reflect the current tech lead and revised the description to include additional information about the project’s functionalities and benefits.
Why was it changed:
To ensure accurate information about project leadership and provide a clearer, more comprehensive overview of the project's capabilities and advantages.
Testing Done:
cross-checked the content on the portfolio page by running in localhost.

### CODE REVIEW

File: docs/project_meltshiny/about.md
Comment (kungfuchicken) [inline docs/project_meltshiny/about.md:1]: The names after anyone currently in capstone should be in alphabetical order.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
Aside from the very minor name sorting issue, these all look great! Your updates to the User Guide are significant improvements in clarity. Your updates to Technical information are also good.

**Review by kungfuchicken — CHANGES_REQUESTED** [review]
I'm concerned that this PR has pulled in changes for multiple projects, and many (most?) of the changes are already in production. At this point I don't trust that this PR will make safe changes, and I don't have the bandwidth to pick through what should stay or go. Does it make sense to close this PR and restart it from scratch?

**Comment by BollepalliPranitha** [comment]
@kungfuchicken Sure. Created a New PR only with the required changes #149 . You can close this PR

**Pull request closed without merging**